project/plugins.sbt

```diff
@@ -1,14 +1,12 @@
 resolvers ++= DefaultOptions.resolvers(snapshot = true)
-resolvers ++= Seq(
-  Resolver.typesafeRepo("releases"),
-  Resolver.sonatypeRepo("releases"),
-  Resolver.sonatypeRepo("snapshots"), // used by deploy nightlies, which publish here & use -Dplay.version
-)
+resolvers ++= Resolver
+  .sonatypeOssRepos("snapshots") // used by deploy nightlies, which publish here & use -Dplay.version
 
-addSbtPlugin("com.typesafe.play" % "interplay"            % sys.props.getOrElse("interplay.version", "3.1.6"))
 addSbtPlugin("com.typesafe.play" % "sbt-plugin"           % sys.props.getOrElse("play.version", "2.9.0-M7"))
 addSbtPlugin("com.typesafe.play" % "play-docs-sbt-plugin" % sys.props.getOrElse("play.version", "2.9.0-M7"))
 
 addSbtPlugin("org.scalameta"     % "sbt-scalafmt"    % "2.5.2")
 addSbtPlugin("com.typesafe"      % "sbt-mima-plugin" % "1.1.3")
 addSbtPlugin("de.heikoseeberger" % "sbt-header"      % "5.10.0")
+
+addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.5.12")
```

Comment:
@mkurz How did we release this library without `sbt-ci-release` plugin? 🤔

Reply:
interplay provided that: https://github.com/playframework/interplay/blob/ad1bddb900e2883f7ad78e59c70fc86603903a52/build.sbt#L19